salt/modules/state.py

```diff
@@ -705,6 +705,12 @@ def apply_(mods=None, **kwargs):
 
             salt '*' state.apply localconfig=/path/to/minion.yml
 
+    state_events
+        The state_events option sends progress events as each function in
+        a state run completes execution.
+
+        .. versionadded:: 3006.0
```

Comment:
**question (blocking):** this one is for @Ch3LL , @garethgreenaway , @whytewolf , or anyone else that knows the state of the 3006.0 freeze... can this one sneak in?

Reply:
@nicholasmhughes We should be able to get this in for 3006.

Reply:
Great :)